playground/backend/internal/preparers/java_preparators_test.go

```diff
@@ -88,25 +88,39 @@ func Test_replace(t *testing.T) {
 	}
 }
 
-func TestGetJavaPreparators(t *testing.T) {
+func TestGetJavaPreparers(t *testing.T) {
 	type args struct {
-		filePath string
+		filePath   string
+		isUnitTest bool
+		isKata     bool
 	}
 	tests := []struct {
 		name string
 		args args
 		want int
 	}{
 		{
-			name: "all success",
-			args: args{"MOCK_FILEPATH"},
-			want: 4,
+			name: "Test number of preparers for code",
+			args: args{"MOCK_FILEPATH", false, false},
+			want: 2,
+		},
+		{
+			name: "Test number of preparers for unit test",
+			args: args{"MOCK_FILEPATH", false, false},
+			want: 2,
+		},
+		{
+			name: "Test number of preparers for kata",
+			args: args{"MOCK_FILEPATH", false, false},
```

Comment:
Seems like you should put true as a third argument

Reply:
Done